### PR TITLE
ADD: Accessibility button trait to TouchableOpacity

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/cxt5h2e9hnvkk1kyzvhdf3afm8?device=pixel4

[download apk](http://206.189.62.190:1488/accessbility-1624539903.apk)

File: loc/en.json [hunk 1]
```diff
@@ -186,6 +186,7 @@
     "details_no_signed_tx": "The selected file doesn’t contain a transaction that can be imported.",
     "details_note_placeholder": "Note to Self",
     "details_scan": "Scan",
+    "details_scan_hint": "Double tap to scan or import a destination",
```
Comment (Overtorment) [inline loc/en.json:189]: double tap..?

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**